lib/puppet/type/ec2_instance.rb

```diff
@@ -20,6 +20,10 @@
     desc 'the security groups to associate the instance'
   end
 
+  newparam(:iam_instance_profile) do
```

Comment:
It would be nice if this where a property, as properties are displayed automatically by puppet resource.